src/scene/forward-renderer.js

```diff
@@ -1442,9 +1452,19 @@ Object.assign(pc, function () {
                         }
                         pass++;
                         if (type === pc.LIGHTTYPE_DIRECTIONAL) light._visibleLength[cameraPass] = -1; // prevent light from rendering more than once for this camera
+
+                        // #ifdef DEBUG
+                        this.device.popMarker();
```

Comment:
Does this need a matching if statment about `passes > 1`?

Reply:
fixed